<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="Frame_Principal">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(240, 240, 240);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="Titulo_Encurtador">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Encurtador simples de url com python&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_Vazio_esqA">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_8">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="lbl1_Link_para_encurtar">
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;Arial&quot;;
color: #343434;
</string>
               </property>
               <property name="text">
                <string>Link para encurtar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le0_link_encurtar">
               <property name="styleSheet">
                <string notr="true">border: 1px solid #343434;
font: 9pt &quot;Arial&quot;;
</string>
               </property>
               <property name="placeholderText">
                <string>URL</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl2_link_encurtado">
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;Arial&quot;;
color: #343434;
</string>
               </property>
               <property name="text">
                <string>Link encurtado</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le1_link_encurtado">
               <property name="styleSheet">
                <string notr="true">border: 1px solid #343434;
font: 9pt &quot;Arial&quot;;
</string>
               </property>
               <property name="placeholderText">
                <string>URL</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_vazio_dirA">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame_vazio_esqB">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="frame_9">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item alignment="Qt::AlignTop">
              <widget class="QPushButton" name="btn_Encurtar">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	font: 10pt &quot;Arial&quot;;
	color: #fff;
	outline: 0;
	border: 5px solid #343434;
	background-color: rgb(52, 52, 52);
}

QPushButton:hover {
	color: #000000;
	border: 5px solid #8d8d8d;
	background-color: rgb(141, 141, 141);
}


QPushButton#btn_Encurtar:pressed {
	padding-left: 2px;
	padding-top: 2px;
	color: #000000;
	border: 5px solid #fff;
	background-color: rgb(255, 255, 255);
}</string>
               </property>
               <property name="text">
                <string>Encurtar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_vazio_dirB">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
